qiskit_aqua/algorithms/components/multiclass/error_correcting_code.py

```diff
@@ -21,21 +21,48 @@
 from sklearn.metrics.pairwise import euclidean_distances
 from sklearn.multiclass import _ConstantPredictor
 
+from qiskit_aqua.algorithms.components.multiclass.multiclass_extension import MulticlassExtension
+
 logger = logging.getLogger(__name__)
 
-class ErrorCorrectingCode:
+class ErrorCorrectingCode(MulticlassExtension):
     """
       the multiclass extension based on the error-correcting-code algorithm.
     """
+    ErrorCorrectingCode_CONFIGURATION = {
+        'name': 'ErrorCorrectingCode',
+        'description': 'ErrorCorrectingCode extension',
+        'input_schema': {
+            '$schema': 'http://json-schema.org/schema#',
+            'id': 'error_correcting_code_schema',
+            'type': 'object',
+            'properties': {
+                'estimator_class_name': {
+                    'type': 'string',
+                    'default': 'RBF_SVC_Estimator',
+                    'oneOf': [
+                        {'enum': ['RBF_SVC_Estimator', 'QKernalSVM_Estimator']}
+                    ]
+                },
+                'code_size': {
```

Comment:
is there any constraint for the `code_size`? like >= 1?

Reply:
done